api/Models/Account.cs

```diff
@@ -1,9 +1,9 @@
 ﻿namespace iTechArt.Hotels.Api
 {
-    public partial class Account
+    public class Account
     {
         public int Id { get; set; }
-        public string Email { get; set; }
-        public string Password { get; set; }
+        public virtual string Email { get; set; }
+        public virtual string Password { get; set; }
```

Comment:
Why do you need them virtual?

Reply:
For lazy loading; have already removed them